### ISSUE TITLE
Example transaction is stuck as SUBMITTED

### ISSUE THREAD

**Issue by confuseSUN** [koxu1996/gevulot-risc0-integration#1]
> I checked your tx hash 3c90fd79ea28803f2d826d23b7c332c1c7be50c95e73fa85472a706ac979784a on the dashboard of Geuvlot. Why is it still in the SUBMITTED state?

**Comment by Chandher05** [comment]
Even I am curious as to why, I submitted another transaction and even that is staled at just SUBMITTED state

**Comment by koxu1996** [comment]
Indeed, transaction [3c90...784a](http://dashboard.gevulot.com/tx/3c90fd79ea28803f2d826d23b7c332c1c7be50c95e73fa85472a706ac979784a) is stuck as submitted:

![image](https://github.com/user-attachments/assets/a1e86c66-4e07-41a4-8257-669c845bbea5)

That's because Risc0 prover was crashing with `No such file or directory` when trying to create tempfile in non-existing `/tmp`. I already fixed that by setting `TMPDIR` env - 8fc28001e69c8dc1288902ab1fb09e0b13da0f3e.

---

I redeployed everything - transaction [67aa...2144](http://dashboard.gevulot.com/tx/67aa482daeac5516dbd010b87720f255060ebc678f83041c8cd3caeb056c2144) is proved successfully by 2 nodes :tada:.

However, it gets stuck later in the verification step:

![image](https://github.com/user-attachments/assets/5a516790-351e-4e89-8427-a44204b31d98)

Right now I have no time and interest in debugging this. Why are you interested in this repository?
